kubernetes/config-maps/nginx-storage/default.conf.j2

```diff
@@ -207,14 +208,34 @@ server {
 
             os.rename("/srv/storage/" .. tmp_file_name, "/srv/storage/" .. header_path_with_filename)
             os.remove("/srv/storage/" .. tmp_file_name .. ".shactx")
+
+            -- Send confirmation to the conductor after uploaded file.
+            local http = require "resty.http"
+            local httpc = http.new()
+            local res, err = httpc:request_uri("{{ upload_report_url }}" .. header_path_with_filename, {
+            method = "POST",
+            })
```

Comment:
When splitting code into multiple lines do it like this:

``` lua
local res, err = httpc:request_uri(
    "{{ upload_report_url }}" .. header_path_with_filename,
    {method = "POST"},
)
```

So, each parameter on a separate line and one level of indentation.

Reply:
So, that's not exactly a point. Second argument (dict) contains all parameters. Below correctly version.
```
local res, err = httpc:request_uri(
    "{{ upload_report_url }}" .. header_path_with_filename,
    {
        method = "POST",
    }
)
```
I think that style from documentation is more understandable:
```
local res, err = httpc:request_uri("{{ upload_report_url }}" .. header_path_with_filename, {
    method = "POST",
})
```

Reply:
If there's only one element in a dict, it's fine to put it on a single line.

Those other two versions you gave are also OK. I'd still probably choose mine but If you want to use one of them that's fine too.

Reply:
Done.